src/order/succ_pred.lean

```diff
@@ -236,6 +242,13 @@ def of_succ_le_iff (succ : α → α) (hsucc_le_iff : ∀ {a b}, succ a ≤ b 
   succ_le_of_lt := λ a b, hsucc_le_iff.2,
   le_of_lt_succ := λ a b h, le_of_not_lt ((not_congr hsucc_le_iff).1 h.not_le) }
 
+lemma covers_iff_eq_pred [no_top_order α] [succ_order α] : a ⋖ b ↔ b = succ a :=
```

Comment:
```suggestion
lemma covers_iff_succ_eq [no_top_order α] [succ_order α] : a ⋖ b ↔ succ a = b :=
```

Reply:
Which way should the `succ` or `pred` go? I started putting them on the right but the left sounds like it follows the "harder = simpler" rule better.